firmware/firmware.cpp

```diff
@@ -558,13 +577,12 @@ void update(uint32_t time) {
 // returns address to flash file to
 uint32_t get_flash_offset_for_file(BlitGameHeader &bin_header) {
 
-  // temporary load address for working on multiple app support without PIC being ready
-  // in future this will probably be more of a "find free space" function
   if(bin_header.magic == blit_game_magic) {
-    auto expected_addr = bin_header.start;
+    // TODO: handle full
```

Comment:
Handle full what? 😆

Reply:
I think that was about the flash being full. Currently it just fails if you manage to do that...